xml/System/Environment+SpecialFolder.xml

```diff
@@ -539,7 +539,7 @@
       </ReturnValue>
       <MemberValue>23</MemberValue>
       <Docs>
-        <summary>A folder for components that are shared across applications. This special folder is valid only for Windows NT, Windows 2000, and Windows XP systems. Added in the .NET Framework 4.</summary>
+        <summary>A folder for components that are shared across applications. This special folder is valid only for Windows NT, Windows 2000, and Windows XP systems. Added in .NET Framework 4.</summary>
```

Comment:
I think this was meant to imply "starting with WinNT / 2000 / XP", so it would encompass all in-market versions of Windows.